app/src/main/java/com/example/nidoqueue/ExperimentCreateFragment.java

```diff
@@ -93,17 +93,17 @@ public void onClick(DialogInterface dialog, int which) {
                         String typeSelected = type.getSelectedItem().toString();
                         Boolean geoLocationChecked = geoLocation.isChecked();
 
-                        if (typeSelected.equals("Count")) {
-                            listener.onOkPressed(new ExpCount(dbManager.getUser(), expName, expDesc, geoLocationChecked), typeSelected);
-                        } else if (typeSelected.equals("Binomial")) {
-                            listener.onOkPressed(new ExpBinomial(dbManager.getUser(), expName, expDesc, geoLocationChecked), typeSelected);
-                        } else if (typeSelected.equals("Non Negative")) {
-                            listener.onOkPressed(new ExpNonNegative(dbManager.getUser(), expName, expDesc, geoLocationChecked), typeSelected);
-                        } else if (typeSelected.equals("Measurement")) {
-                            listener.onOkPressed(new ExpMeasurement(dbManager.getUser(), expName, expDesc, "", geoLocationChecked), typeSelected);
-                        } else {
-                            Toast.makeText(getContext(), "Please select experiment type", Toast.LENGTH_SHORT).show();
-                        }
+//                        if (typeSelected.equals("Count")) {
+//                            listener.onOkPressed(new ExpCount(dbManager.getUser(), expName, expDesc, geoLocationChecked), typeSelected);
+//                        } else if (typeSelected.equals("Binomial")) {
+//                            listener.onOkPressed(new ExpBinomial(dbManager.getUser(), expName, expDesc, geoLocationChecked), typeSelected);
+//                        } else if (typeSelected.equals("Non Negative")) {
+//                            listener.onOkPressed(new ExpNonNegative(dbManager.getUser(), expName, expDesc, geoLocationChecked), typeSelected);
+//                        } else if (typeSelected.equals("Measurement")) {
+//                            listener.onOkPressed(new ExpMeasurement(dbManager.getUser(), expName, expDesc, "", geoLocationChecked), typeSelected);
+//                        } else {
+//                            Toast.makeText(getContext(), "Please select experiment type", Toast.LENGTH_SHORT).show();
+//                        }
```

Comment:
Please delete this code instead of commenting it out, that way no one has to go back and delete it later.